ios/Classes/BannerDelegate.swift

```diff
@@ -8,24 +8,24 @@
 import Appodeal
 import Foundation
 
-extension SwiftAppodealFlutterPlugin: AppodealBannerDelegate {
-    public func bannerDidLoadAdIsPrecache(_: Bool) {
+extension SwiftAppodealFlutterPlugin: APDBannerViewDelegate {
```

Comment:
Aha, I noticed that I was using `AppodealBannerDelegate` before but to be honest I don't remember where I took that from.

In any case I consulted the documentation and it seems that `BannerViewDelegate` is the recommended way to track banner events, but I don't see any real advantage of one over the other. I agree to leave the implementation in the way you put here, but do you have more context why we should use `APDBannerViewDelegate` instead? I'm just curious 🙂

One more thing, it seems that we also have an alias for `APDBannerViewDelegate` called `AppodealBannerViewDelegate`. Can you use `AppodealBannerViewDelegate` instead since it's a little more descriptive and "swifty" than `APDBannerViewDelegate`? 😅

Reply:
The reason we changed it is that we're now using `APDBannerView` instead of the old `banner()` implementation, which only works with `APDBannerViewDelegate`. Not really sure if there is really any difference though 🙂 Will change the name to be more "swiftly" 👍